atd-vze/src/views/Reports/crashesInconsistentTableParameters.js

```diff
@@ -0,0 +1,67 @@
+/*
+* Configuration for the table view, no need for advanced filters.
+* */
+
+export const crashGridTableColumns = {
+  atc_crash_id: {
+    primary_key: true,
+    searchable: true,
+    sortable: true,
+    label_search: "Search by Crash ID",
+    label_table: "Crash ID",
+    type: "Int",
+  },
+  atc_death_cnt: {
+    searchable: false,
+    sortable: true,
+    label_search: "",
+    label_table: "Crash Death Count",
```

Comment:
Might be good to clarify that the death count for crashes is the CRIS Death Count as opposed to APD Confirmed. Just a thought.